src/CombFilter/CombFilter.h

```diff
@@ -0,0 +1,61 @@
+#if !defined(__CombFilter_hdr__)
+#define __CombFilter_hdr__
+
+#include "CombFilterIf.h"
+#include "RingBuffer.h"
+
+class CCombFilterBase {
+public:
+    CCombFilterBase(float fMaxDelayLengthInS, float fSampleRateInHz, int iNumChannels);
+    virtual ~CCombFilterBase();
+
+    virtual Error_t process(float** ppfInputBuffer, float** ppfOutputBuffer, int iNumberOfFrames) {
```

Comment:
Not sure, could be neater to just make this mandatory override function = 0?

Reply:
Yes! I have found such pure virtual function definition in other's codebase and I think it's a very good design! I guess I have thought about this and later forgot about it. Will add soon, thanks!